lib/shopify_api.rb

```diff
@@ -32,3 +32,8 @@ module ShopifyAPI
 else
   require 'active_resource/connection_ext'
 end
+
+if ENV["SHOPIFY_LOG"] == '1'
+  ActiveResource::Base.logger = Logger.new("shopify_api.log")
```

Comment:
Should the env var be a path instead to make this easier to use in production? `Logger.new(ENV['SHOPIFY_LOG'])`

Reply:
We could do that!